utils/dbfixup.py

```diff
@@ -154,6 +153,12 @@ def add_zero_bits(
                     zero_range(tag, bits, 22, 25)
                 zero_groups(tag, bits, zero_db, strict=strict, verbose=verbose)
 
+                if len(bits) == 0:
+                    verbose and print(
```

Comment:
Should this be ERROR, not WARNING? A WARNING to me is something that wasn't given enough data. In this case the database is actually oversolved and thus invalid

Reply:
It can be an ERROR on strict, I agree.  However, when the zero db is missing entries, treating this like an error makes it hard to boot strap a fuzzer.